Relax version requirements and enable build for older versions of py-setuptools.

--- pyproject.toml.orig	2025-09-26 22:17:55 UTC
+++ pyproject.toml
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools >= 77.0.3", "setuptools-scm ~= 7.0"]
+requires = ["setuptools", "setuptools-scm"]
 build-backend = "setuptools.build_meta"
 
 [project]
@@ -9,8 +9,7 @@ dynamic = ["version"]
 requires-python = ">= 3.9"
 dynamic = ["version"]
 
-license = "BSD-3-Clause"
-license-files = ["LICENSE.rst"]
+license = { text="BSD-3-Clause" }
 readme = "README.rst"
 
 dependencies = [
